angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -5,23 +5,28 @@
         <a>All ({{ totalCount }})</a>
       </li>
       <li ng-click="filterTo('myuw')" id="myuw-selector">
-        <a>MyUW ({{myuwFilteredResults.length}})</a>
+        <a>MyUW <span ng-hide="portletListLoading">({{myuwFilteredResults.length}})</span>
+        <i ng-show="portletListLoading" class="fa fa-spinner fa-pulse"></i></a>
       </li>
       <li ng-click="filterTo('directory')" id="directory-selector">
-        <a>Directory ({{wiscDirectoryTooManyResults ? '25+' : wiscDirectoryResultCount}})</a>
+        <a>Directory <span ng-hide="directorySearchLoading">({{wiscDirectoryTooManyResults ? '25+' : wiscDirectoryResultCount}})</span>
+        <i ng-show="directorySearchLoading" class="fa fa-spinner fa-pulse"></i></a>
       </li>
       <li ng-click="filterTo('google')" id="google-selector">
-        <a>Wisc.edu ({{googleResultsEstimatedCount}})</a>
+        <a>Wisc.edu <span ng-hide="googleSearchLoading">({{googleResultsEstimatedCount}})</span>
+        <i ng-show="googleSearchLoading" class="fa fa-spinner fa-pulse"></i></a>
       </li>
     </ul>
   </div>
 
-  <div id="myuw-results" class='search-results-container'>
-    <h4 class="header">MyUW</h4>
-    <hr>
+  <div id="myuw-results" class='search-results-container'
+    ng-show="myuwFilteredResults.length > 0 
```

Comment:
I'm uneasy about this.  I think we may have talked about this always being there.  I'm open to being wrong on that.